OxfordBodleian/Ullendorff/BDLeu1.xml

```diff
@@ -140,6 +417,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         </item>
                                     </list>
                                 </collation>
+                                <condition key="deficient">The borders of the leaves are darkened by use. Small stains throughout. The upper margin shows traces of
+                                humidity (e.g. <locus from="134v" to="135r"/>). The binding is not original; leaves are probably partly detached.</condition>
```

Comment:
I think that the wooden boards are original. It seems that also the original thread was reused to sew the book. The quarter cover and the guards have been definitely added later. Do these consideration go in the bindingDesc part?

Reply:
Yes yes, it belongs to bindingDesc